### ISSUE TITLE
remove holoviews dependency

### ISSUE THREAD

**Issue by chanana** [enveda/mspaint#8]
I don't love it

**Comment by jtprince** [comment]
Yeah, if the abstraction level of holoviews is not right, then it makes sense to drop it down one notch.

**Comment by chanana** [comment]
it's really cool as a concept but in practice there's things that become enough of an edge case that we could drop it potentially.

1. you can save to svg if you have a matplotlib or bokeh plot but not a plotly plot. However, plotly shows you how to save the svg separately using the `kaleido` package.
2. you can set an image size if you're using plotly or bokeh but not if you're using matplotlib. Obviously, matplotlib on its own does have that capability.
3. plotting a scatter plot as Scatter vs Points has subtleties that I need to get acquainted with (in holoviews) and can result in differences in how you interact with the plot for the different engines

These were the things I ran into so far. The other solution here would be to attempt to put PRs upstream in holoviews itself so we can use it more consistently.
